src/main/java/net/imagej/ops/filter/tubeness/TubenessOp.java

```diff
@@ -0,0 +1,14 @@
+package net.imagej.ops.filter.tubeness;
+
+import net.imagej.ops.Ops;
+import net.imagej.ops.special.hybrid.UnaryHybridCF;
+import net.imglib2.IterableInterval;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.type.numeric.RealType;
+import net.imglib2.type.numeric.real.DoubleType;
+
+public interface TubenessOp< T extends RealType< T > > extends Ops.Filter.Tubeness,
```

Comment:
If I got the idea of `Ops.list` and `Ops.vm` correctly, this interface should not be necessary, as an interface is automatically created at `Ops.Filter.Tubeness` using template parsing. So if you change your `DefaultTubeness` annotation to `@Plugin( type = Ops.Filter.Tubeness.class )` it should work.

Reply:
Other ops in the filter namespace have the same hierarchy. They have the `Tubeness` interface in `Ops.list` and the mother `TubenessOp` that extends it AND specifies the type of ops (hybridy, unary, etc) in the code package.

Reply:
Oh, I see. I didn't fully understand the hierarchy yet, sorry for the buzz then.

Reply:
Ok. Let's simplify then.